qulice-findbugs/src/main/java/com/qulice/findbugs/FindBugsValidator.java

```diff
@@ -176,14 +176,22 @@ private String excludes(final Environment env,
      * Creates xml with exclude patterns in findbugs native format.
      * @param excludes Exclude patterns
      * @return XML with findbugs excludes
+     * @todo #350 It's better to build XML using Xembly, but when it's
```

Comment:
@longtimeago I've used version 0.19.2
I'll try to push code with Xembly so you can check logs and correctness of usage